newlib/libc/include/errno.h

```diff
@@ -37,6 +37,10 @@ SUCH DAMAGE.
 #include <sys/cdefs.h>
 
 #if __STDC_WANT_LIB_EXT1__ == 1
+#ifndef __STDC_LIB_EXT1__
```

Comment:
Should this be `#undef __STDC_LIB_EXT1__` instead of only setting the value if not already set?

Reply:
Ah, ok. Thanks for trying. We'll go with this version.